src/index.js

```diff
@@ -16,14 +16,53 @@ type PropsType = {
     quotes?: string,
     highlight?: (str: string, lang: string) => string,
   },
+  rules?: {
+    block?: {
+      enable?: array,
+      disable?: array,
+    },
+    core?: {
+      enable?: array,
+      disable?: array,
+    },
+    inline?: {
+      enable?: array,
+      disable?: array,
+    },
+  },
 };
 
 const MarkdownRenderer = ({
   markdown,
   options: { preset, ...options } = {},
+  rules,
   ...props
 }: PropsType) => {
   const remarkable = new Remarkable(preset || 'default', options);
+
+  /* rule is an object used to enable/disable rules.
+    {
+      core: {
+        enable: [ 'abbr' ]
+      },
+      inline: {
+        enable: [ 'ins', 'mark' ]
+      },
+      block: {
+        disable: [ 'table', 'footnote' ]
+      }
+    }
+  */
+  if (rules) {
+    Object.keys(rules).forEach(rule => {
+      Object.keys(rules[rule]).forEach(flag => {
+        if (remarkable[rule]) {
+          remarkable[rule].ruler[flag](rules[rule][flag]);
+        }
+      });
+    });
+  }
```

Comment:
Not a big deal at all, but would it make sense to move the `if` statement to be closer to the `rule` forEach. Sorta failing sooner...

```
if (rules) {
   Object.keys(rules).forEach(rule => {
      if (remarkable[rule]) {
         Object.keys(rules[rule]).forEach(flag => {
             remarkable[rule].ruler[flag](rules[rule][flag]);
         });
      }
    });
 }
```